docs/test-config.md

```diff
@@ -192,6 +194,8 @@ configurations:
       run_time: str # (4)!
       # Test target URL 
       target_host: str # (5)!
+      # * Test stop timeout
+      stop_timeout: str
```

Comment:
need to align with code and follow whatever changes based on my previous comment. Moreover, if you can kindly add an annotation # (11)! and explain in more words what it does and to what _locust_ argument it maps. This is similar to all the other options.